spec/factories/fitbit_activities.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :fitbit_activity do
+    steps 100
+    floors  1
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.